slack/rtm/client.py

```diff
@@ -384,11 +390,20 @@ async def _read_messages(self):
                 )
                 self._websocket = None
                 await self._dispatch_event(event="close")
+                num_of_running_callbacks = len(text_message_callback_executions)
+                if num_of_running_callbacks > 0:
+                    self._logger.info(
+                        "WebSocket connection has been closed "
+                        f"though {num_of_running_callbacks} callback executions were still in progress"
```

Comment:
This is just a msg for the devs that they lost some executions by running close? Guessing we wouldn't want to halt the close event

Reply:
Yes, as you guessed, this message is meant for giving a bit more information for developers for the case they unexpectedly encounter this situation. This message indicates that if some of the remaining tasks are going to send messages back to the Slack server over a WebSocket connection, it won't work. If the remaining tasks don't talk to the Slack server, they're not terminated yet, so that they may result in success.